Tests/Functional/Datahandler/ContentDefender/DefaultLanguageTest.php

```diff
@@ -224,4 +224,35 @@ public function copyElementIntoContainerAtTopCopyAllowedCTypeElement(): void
         self::assertSame(1, (int)$row['tx_container_parent'], 'element should  be inside container');
         self::assertSame(200, (int)$row['colPos'], 'element should  be inside container colPos');
     }
+
+    /**
+     * @test
+     * @group content_defender
+     */
+    public function copyChildFromOtherContainerIntoColposWhereTargetElementInOtherColposHasRestrictionIsAllowd(): void
+    {
+        $this->importCSVDataSet(ORIGINAL_ROOT . 'typo3conf/ext/container/Tests/Functional/Datahandler/ContentDefender/Fixtures/DefaultLanguage/copy_child_from_other_container_into_colpos_where_target_element_in_other_colpos_has_restriction.csv');
```

Comment:
I think we should avoid ORIGINAL_ROOT and use __DIR__

Reply:
:+1: